app.js

```diff
@@ -25,38 +25,38 @@ headSelect.addEventListener('change', () => {
 });
 
 middleSelect.addEventListener('change', () => {
-    // > set the character middle from the value of the middle <select>
-    // > redisplay the character
+    character.middle = middleSelect.value;
+    displayCharacter();
```

Comment:
setting state based on user interaction then using that state to update the view